core-rust/node-common/src/jni/addressing.rs

```diff
@@ -158,6 +158,24 @@ extern "system" fn Java_com_radixdlt_identifiers_Address_nativeVirtualAccountAdd
     )
 }
 
+#[no_mangle]
+extern "system" fn Java_com_radixdlt_identifiers_Address_nativeVirtualIdentityAddress(
+    env: JNIEnv,
+    _class: JClass,
+    request_payload: jbyteArray,
+) -> jbyteArray {
+    jni_sbor_coded_call(
+        &env,
+        request_payload,
+        |unchecked_public_key_bytes: [u8; Secp256k1PublicKey::LENGTH]| {
```

Comment:
How about ed25519?

Reply:
I only added this method for test purposes (I need to obtain some identity address to test stuff with), and I arbitrarily chosen Secp256k1.
(in a Rest API-level test I don't aim to cover all cases)